iRonCub-Mk3/hardware/electronics/face-eb22-j0_1-eln.xml

```diff
@@ -22,7 +22,7 @@
                 <param name="maxTimeOfRXactivity">      400                 </param>
                 <param name="maxTimeOfDOactivity">      300                 </param>   
                 <param name="maxTimeOfTXactivity">      300                 </param>                
-                <param name="TXrateOfRegularROPs">      2                   </param> 
+                <param name="TXrateOfRegularROPs">      5                   </param> 
```

Comment:
Why are you updating the frequency of the face joints, but not the head, legs and torso joints?

Mostly we won't need the face joints to run at higher frequency, unless you want all the values to be consistent.

Reply:
Because the head is already at 200 Hz, which is enough since we are not going to really use it; I decreased the frequency of the head because we don't need it to run that fast.